mautrix_telegram/__main__.py

```diff
@@ -112,7 +115,10 @@ def prepare_bridge(self) -> None:
 
         if self.config['metrics.enabled']:
             self.as_connection_metric_task = self.loop.create_task(self._loop_check_as_connection_pool())
-        
+
+        if self.config['telegram.liveness_timeout'] and self.config['telegram.liveness_timeout'] >= 1:
```

Comment:
Does this config item exist even if it's not defined? Otherwise you will hit a `KeyError`. Can't remember if the `copy()` below in the config python side actually sets it or copies it if it's set.

Anyway, to be safe, `.get()` pattern is always better and safer.

```suggestion
        if self.config.get('telegram.liveness_timeout') and self.config['telegram.liveness_timeout'] >= 1:
```

Reply:
Could we simplify and do `self.config.get('telegram.liveness_timeout', 0) >= 1`?

Reply:
Yeah good point :)